Makefile

```diff
@@ -53,11 +53,26 @@ utils:
 	$(MAKE) -C $(HTTP_PARSER_PATH) http_parser.o
 test: utils
 	$(MAKE) -C $(TEST_PATH)
-%.o:%.c
+%.o:%.c cert
 	$(CC) -v -c $(CFLAGS) $(INCLUDES) -MMD -MF $@.d -o $@ $<
 
 hal:
 	$(MAKE) -C $(HAL_PATH)
+cert: check_pem
+	@xxd -i $(PEM) > connectivity/ca_cert.inc
```

Comment:
What is the default string for `$(PEM)`?

Reply:
`$(PEM)` is the certificate path. It should be provided as following command. `make PEM=ca.pem`. If the `$(PEM)` is not set, it will raise error.

Reply:
Let's set default and preserved path, otherwise it would raise problems with CI and general use.